pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler_test.go

```diff
@@ -179,6 +181,7 @@ func TestCreateDefaultDeployment(t *testing.T) {
 					},
 					Annotations: map[string]string{
 						"annotation": "annotation-value",
+						"service.beta.openshift.io/serving-cert-secret-name": "default-predictor-serving-cert",
```

Comment:
we use this annotation key many places so why don't we add this to constants ?
```service.beta.openshift.io/serving-cert-secret-name```

Reply:
done